dallinger/recruiters.py

```diff
@@ -375,6 +376,7 @@ class ProlificRecruiter(Recruiter):
     """A recruiter for [Prolific](https://app.prolific.com/)"""
 
     nickname = "prolific"
+    publish_experiment_default = False
```

Comment:
You can now remove `publish_experiment_default = False`

Reply:
Will `publish_experiment_default` in  `ProlificRecruiter` be set correctly? I am not clear on the order of execution. Hmmm

Reply:
Yes, i think everything in the parent Recruiter class should get run first!

Reply:
But then `publish_experiment_default = not supports_delayed_publishing` will only be executed once with `supports_delayed_publishing = False`. How can then `publish_experiment_default` be ever overridden for `ProlificRecruiter` ? Sorry if I am stupid here.

Reply:
Ah, no, you are being clever here (or I'm being stupid!)

You're right, of course. So we would need to make `publish_experiment_default` a property, i.e.

```py
@property
def publish_experiment_default(self):
    return not supports_delayed_publishing

Reply:
OK, I thought about doing something similar. But this is the best solution.
I hope we got it now. Made a commit.

Reply:
Waiting for your go...

Reply:
Sorry, yes -- approved!